src/tasks/AutoHeistTask.py

```
@@ -81,72 +92,169 @@ def run(self):
+        if BoxList:
+            for i, box in enumerate(BoxList):
+                text = getattr(box, 'name', '')
+                if not text:
+                    continue
+                if "本局收益" in text:
+                    try:
+                        if i+1 < len(BoxList):
+                            next_box_text = BoxList[i+1].name
+                            clean_num = re.sub(r'[^\d]', '', next_box_text)
+                            earn_fcash = int(clean_num)
+                    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)